### ISSUE TITLE
Sodium Option and Sodium Extra features from the latest Embeddium.

### ISSUE THREAD

**Issue by erialsan** [Asek3/sodium-1.12#53]
### Request Description

This is the implementation of the Sodium Option and Sodium Extra features from the latest Embeddium.
Although we may not be aware of the latest Embeddium situation as this project has not been running for several months, they allow for more configurations.
I don't have the skills to tweak this to work with 1.12.2, so I can't do it until a pull request, so I'm leaving it here.

**Comment by duckopia** [comment]
The Sodium Extra is don't have any impact on Minecraft FPS and Performance. So, you don't need it.

**Comment by Garb-in-PacK** [comment]
is there already builded version of this sodium port?

**Comment by duckopia** [comment]
> is there already builded version of this sodium port?

Yes. You can download it here: https://github.com/Asek3/sodium-1.12/actions/runs/9866213559
![{4C524B52-D7F6-4165-8242-2976917977E8}](https://github.com/user-attachments/assets/6474edde-b293-4c2f-b03f-0f84597f303b)
Look at Artifacts
Hope this help you!
